engine/test-utils/src/main/java/io/deephaven/engine/testutil/ControlledUpdateGraph.java

```diff
@@ -1,13 +1,14 @@
 package io.deephaven.engine.testutil;
 
 import io.deephaven.engine.updategraph.impl.PeriodicUpdateGraph;
+import io.deephaven.util.thread.ThreadInitializationFactory;
 
 // TODO (deephaven-core#3886): Extract test functionality from PeriodicUpdateGraph
 public class ControlledUpdateGraph extends PeriodicUpdateGraph {
-
-    public static final ControlledUpdateGraph INSTANCE = new ControlledUpdateGraph();
-
-    private ControlledUpdateGraph() {
-        super("TEST", true, 1000, 25, -1);
+    public ControlledUpdateGraph(String name, boolean allowUnitTestMode, long targetCycleDurationMillis,
```

Comment:
Probably should restore the default ctor, but I think it makes sense to move the singleton instance?